test/cases/api.sh

```diff
@@ -524,8 +534,12 @@ function Test_verifyComposeResultGCP() {
     --zone="$GCP_REGION-a" \
     --image-project="$GCP_IMAGE_BUILDER_PROJECT" \
     --image="$GCP_IMAGE_NAME" \
+    --labels=gitlab-ci-test=true \
     --metadata-from-file=ssh-keys="$GCP_SSH_METADATA_FILE"
   HOST=$($GCP_CMD compute instances describe "$GCP_INSTANCE_NAME" --zone="$GCP_REGION-a" --format='get(networkInterfaces[0].accessConfigs[0].natIP)')
+  
+  echo $GCP_INSTANCE_NAME
+  echo $HOST
```

Comment:
These should be clear from the `-x` flag no?

Reply:
That's true, I just discovered the test fails before creating the instance